content/sensu-go/6.0/sensuctl/back-up-recover.md

```diff
@@ -129,24 +129,55 @@ When you export users, required password attributes are not included.
 You must add a [`password_hash`](../#generate-a-password-hash) or `password` to `users` resources before restoring them with the `sensuctl create` command.
 {{% /notice %}}
 
-## List types of supported resources
+## Supported resource types
 
 {{% notice important %}}
 **IMPORTANT**: The `sensuctl describe-type` command deprecates `sensuctl dump --types`.
 {{% /notice %}}
 
-Use `sensuctl describe-type` to list the types of supported resources.
-For example, to list all types:
+Use `sensuctl describe-type all` to retrieve the list of supported `sensuctl dump` resource types.
+
+{{% notice note %}}
+**NOTE**: The resource types with no short name listed are [commercial features](../../commercial/).
+{{% /notice %}}
```

Comment:
While this is technically true now, it's not necessarily true for the future (for example if a new OSS resource was created). I would rephrase this to something along the lines of `short names are only supported fore core/v2 resources` since that's how it's implemented logically in our code.